public/content/lessons/2016/cross-products/index.mdx

```diff
@@ -1,10 +1,253 @@
 source: _2016/eola/chapter8p2.py
 credits:
 - Lesson by Grant Sanderson
+- Text adaptation by Kurt Bruns
 ---
+
+> "Every dimension is special."
+>
+> \- Jeff Lagarias
+
+Last chapter, I talked about the dot product, showing both the standard introduction to the topic, as well as the deeper view of it related to linear transformations. I'd like to do the same for cross products, which also have a standard introduction along with a deeper understanding in the light of linear transformations, but this time I'm dividing it into two chapters.
+
+Here, I'll try to hit the main points that students are usually shown about the cross product, and in the next chapter I'll show a view which is less commonly taught, but really satisfying when you learn it.
+
+## Two dimensions
+
+We'll start in two dimensions. If you have two vectors, $\vec{\mathbf{v}}$ and $\vec{\mathbf{w}}$, think about the parallelogram that they span out. What I mean by this is that if you take a copy of $\vec{\mathbf{v}}$, and move its tail to the tip of w, and you take a copy of $\vec{\mathbf{w}}$, and move its tail to the tip of $\vec{\mathbf{v}}$, the four vectors enclose a certain parallelogram.
+
+<Figure
+    image="./figures/two-dimensions/AreaOfParallelogram.svg" 
+/>
+
+The cross product of $\vec{\mathbf{v}}$ and $\vec{\mathbf{w}}$, written with the x-shaped multiplication symbol ($\times$), is the area of this parallelogram.
+
+<Figure
+    image="./figures/two-dimensions/AreaOfParallelogramFormula.svg" 
+/>
+
+Well, almost, we also need to consider orientation. Basically, if $\vec{\mathbf{v}}$ is on the right of $\vec{\mathbf{w}}$, then $\vec{\mathbf{v}}$ cross $\vec{\mathbf{w}}$ is positive and equal to the area of that parallelogram.
+
+<Figure
+    image="./figures/two-dimensions/AreaOfParallelogramPositive.svg" 
+/>
+
+But if $\vec{\mathbf{v}}$ is on the left of $\vec{\mathbf{w}}$, the cross product is negative 1 times the area of that parallelogram.
+
+<Figure
+    image="./figures/two-dimensions/AreaOfParallelogramNegative.svg" 
+/>
+
+Notice, this means order matters. If you swap $\vec{\mathbf{v}}$ and $\vec{\mathbf{w}}$, and instead consider $\vec{\mathbf{w}}$ cross $\vec{\mathbf{v}}$, the cross product becomes the negative of whatever it was before.
+
+$$
+\vec{\mathbf{v}} \times \vec{\mathbf{w}} = -\vec{\mathbf{w}} \times \vec{\mathbf{v}}
+$$
+
+The way I always remember this is that when you take the cross product of the two basis vectors in order, i hat cross j hat, the result should be positive. In fact, the order of your basis vectors is what defines orientation. So since i hat is on the right of j hat, I remember that v cross w is positive whenever w is on the right of v.
+
+<Figure
+    image="./figures/two-dimensions/BasisVectorsSign.svg" 
+/>
+
+So for example, with the vectors shown here, I'll just tell you that the area of that parallelogram is 7. And since v is on the left of w, the cross product should be negative, so v cross w is -7.
```

Comment:
```suggestion
For example, with the vectors shown here, the area of that parallelogram is 7 (You'll see how to compute this below). Because $\mathbf{w}$ is a counterclockwise rotation away from $\mathbf{v}$, the cross product should be negative, so $\mathbf{v} \times \mathbf{w} = -7$.
```

Reply:
Since it's been flipped a couple of times in my head and now for the lesson. Should this read "clockwise" and not "counterclockwise"?